components/schedule/WeekNavigator.tsx

```diff
@@ -2,27 +2,26 @@ import LoadingButton from "@mui/lab/LoadingButton";
 import { ArrowBack, ArrowForward } from "@mui/icons-material";
 import { Button, Stack, Typography } from "@mui/material";
 import React, { Dispatch, SetStateAction, useState } from "react";
-import { SitWeekSchedule } from "../../types/integration/sit";
-import { DateTime } from "luxon";
-import { RezervoSchedule } from "../../types/rezervo";
+import { RezervoSchedule, RezervoWeekSchedule } from "../../types/rezervo";
+import { getDateTime } from "../../lib/integration/common";
+
+function getWeekNumber(weekSchedule: RezervoWeekSchedule): number {
+    return getDateTime(weekSchedule[0]!.date).weekNumber;
```

Comment:
Consider handling empty week

Reply:
In theory, this should not be possible, unless one of our integrations returns null values instead of empty arrays for empty weeks. However, I decided to add a verification clause in `fetchRezervoWeekSchedule`.  For reference: https://github.com/mathiazom/sit-rezervo-web/pull/39/commits/d738bd56ce0c25a0623bf257d50e5b5858cb9e5a